packages/@uppy/dashboard/src/components/FileCard/index.jsx

```diff
@@ -59,6 +59,8 @@ class FileCard extends Component {
   }
 
   handleCancel = () => {
+    const file = this.props.files[this.props.fileCardFor]
```

Comment:
should we align this with `uninstall`? I'm not sure where `fileCardFor` comes from so perhaps this is safer
```suggestion
    const { currentImage } = this.getPluginState()
    const file = this.uppy.getFile(currentImage.id)
```